### DISCUSSION TITLE
Adding Air as a Species for GSI

### DISCUSSION THREAD

**Discussion by aobaskaya** [mutationpp/Mutationpp#246] [Q&A]
I was wondering if one could add "Air" as a species to use with GSI. (Not to react with the GSI species, just to be present as the ambient gas.) (Instead of having N2, O2.)
Would this be possible if thermo data, species data and collisions data are supplied? Asking here to make sure what I end up with will be physically consistent. What else would I need to modify?
Thank you in advance.

**Comment by mgoodson-cvd** [comment]
I can't think of anything else. I've done something similar before, creating a "duplicate" species (e.g., "N2_copy") for testing purposes by adding it in three places (the three you mention):

1. my thermo database of choice (e.g., NASA-9),
2. `data/thermo/elements.xml` for the molecular weight, and
3. `data/transport/collisions.xml` for collision data with whatever else is present in my mixture.

The NASA-9 database in Mutation++ already contains `Air` from Gordon and McBride.

**Comment by aobaskaya** [comment]
Thank you for your answer Matthew.
Expanding on the discussion, how could I provide data in `collisions.xml` for air? 
Does anyone know some representative dipole polarizability and effective electron values?

**Reply by mgoodson-cvd** [reply]
Depends on how accurate you need to be...

If you're using single-species "Air", you probably don't need it to be too accurate -- you just need _something_; in which case copying the values for N2 isn't a terrible option.

Another option is to compute the actual mixture values (e.g., N2, O2, Ar, CO2 from Gordon and McBride) and curve fit the values you need for "Air".
